packages/store-cache/src/createDatabaseClient.test.ts

```diff
@@ -361,4 +362,30 @@ describe("createDatabaseClient", () => {
     expect(mock).toHaveBeenNthCalledWith(1, [{ set: [{ key, value }], remove: [], table, namespace }]);
     expect(mock).toHaveBeenNthCalledWith(2, [{ set: [], remove: [{ key }], table, namespace }]);
   });
+
+  // Requires bigint keys to be supported by `tuple-database`, see https://github.com/ccorcos/tuple-database/issues/2
+  // For now we serialize bigints to use them as keys, which changes the comparison (eg. 2n < 10n but "2n" > "10n")
+  it.skip("should be possible to filter based on bigint keys", () => {
```

Comment:
this test is skipped for now because this doesn't work yet without support for bigints in `tuple-database` (it is possible to use the bigint values as filter, but the comparison is based on strings rather than bigints)

Reply:
can we add a TODO and/or issue to remind us to come back to this?